bors.toml

```diff
@@ -1,10 +1,16 @@
 status = [
-  "continuous-integration/travis-ci/push",
+  "Node.js CI / tests (12.x) (push)",
```

Comment:
You can use wildcard syntax here: `status = [ 'ci/gitlab/%' ]`
https://bors.tech/documentation/